web/blog/2023-03-03-ten-hard-truths-junior-developers-need-to-hear.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "10 \"Hard Truths\" All Junior Developers Need to Hear"
+authors: [vinny]
+image: /img/hard-truths-junior-devs.png
+tags: [Junior Developers, Tech Career, Reddit, WebDev]
+---
+
+import Link from '@docusaurus/Link';
+import ImgWithCaption from './components/ImgWithCaption'
+
+<br/>
+
+<ImgWithCaption
+    alt="hard truths for junior devs"
+    source="img/hard-truths-junior-devs.png"
+/>
+
+Ok, I have to admit, these aren’t really Truths, but rather some opinions I’ve formed over my journey switching careers from Educator to Developer.
+
+It’s well known at this point that software — especially web — development is a viable option for someone looking for a new career without going the traditional education route. Due to this, and the fact that salaries tend to be very good, I think a portion of people making the switch might be doing it for the wrong reasons. 
+
+And once you get into that career, as a Junior it can often be difficult to know what you should be doing to advance your career. There are a ton of opinions out there (including mine) and juniors tend to develop a lot of misconceptions, as my colleague and I discussed in our recent [Reddit post](https://www.reddit.com/r/webdev/comments/112im2m/senior_devs_what_are_the_most_damaging/) and [follow-up video](/blog/2023/02/21/junior-developer-misconceptions).
+
+So, I put together this list of things you should consider when starting out a career in tech: 
+
+1. 👎 If you’re doing it solely for the money, you’re not gonna make it. True, you don’t need a degree or anyone’s permission to advance in this career, but you need ambition and mental stamina. A genuine interest is needed to maintain them. 
```

Comment:
```suggestion
1. 👎 **If you’re doing it solely for the money, you’re not gonna make it**. True, you don’t need a degree or anyone’s permission to advance in this career, but you need ambition and mental stamina. A genuine interest is needed to maintain them. 
```
I would bold each opening sentence like above, I think that will make it much more "scannable" and easier to read.